beacon-chain/rpc/beacon/validators.go

```diff
@@ -160,6 +162,56 @@ func (bs *Server) ListValidatorBalances(
 	}, nil
 }
 
+// GetValidatorPerformance reports the validator's latest balance along with other important metrics on
+// rewards and penalties throughout its lifecycle in the beacon chain.
+func (bs *Server) GetValidatorPerformance(
```

Comment:
Is this supposed to be in the PR? It's not detailed anywhere.

Reply:
Amended in the title @0xKiwi